app/client/src/widgets/TableWidgetV2/widget/utilities.ts

```
@@ -771,6 +771,14 @@ export const getColumnType = (
     case "boolean":
       return ColumnTypes.CHECKBOX;
     case "string":
+      // Check for HTML first
+      const isHTML = /<[^>]*>/.test(columnValue as string);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)